Assets/MixedRealityToolkit.SDK/Profiles/HoloLens1/DefaultHoloLens1CameraProfile.asset

```diff
@@ -13,6 +13,14 @@ MonoBehaviour:
   m_Name: DefaultHoloLens1CameraProfile
   m_EditorClassIdentifier: 
   isCustomProfile: 0
+  settingsConfigurations:
+  - componentType:
+      reference: Microsoft.MixedReality.Toolkit.CameraSystem.UnityARCameraSettings,
```

Comment:
why does the Hololens 1/2 want the Unity AR data provider for ios/android?

Reply:
it's to allow the demo apps to build without modifying the profile (similar to how the profiles have not had unsupported input modalities removed).

if you feel strongly, we can document how to make the demos work on mobile ar

Reply:
As a new user, it is exceptionally weird to have unique profiles for HL1 and HL2 devices....but not have a DefaultUnityARConfigurationProfile for iOS/Android

I think it makes sense to add Unity AR in the DefaultMRTKConfigurationProfile since that is sort of a catch all for all platforms

Reply:
As much as we can work back to only needing a single profile that's configurable and configured for multiple platforms, the better.

There's definitely still work to be done to make it configurable enough to support that world.

In this case, there shouldn't be any side-effect for HoloLens by including this. It then allows a developer to use this profile and be able to build their app for both devices without changing profiles.

Reply:
Why put it in HL Default profile and not the actual default profile DefaultMRTKConfigurationProfile? 

Isn't there no side-effect really to having the Boundary and Teleport systems enabled in the HL profile...but they are disabled because you won't be using them. Same with this camera profile, why would I use an iOS/android profile in a HL targeted profile?

Reply:
We also have lots of other profiles to demonstrate eye tracking or some other example scene/feature. Why wouldn't we have a AR Foundation profile?

Reply:
I agree it should go into the Default as well

Reply:
Looks like it already is: https://github.com/microsoft/MixedRealityToolkit-Unity/pull/6344/files#diff-ad6f41e1820507363710e270f3007e5a